interfaces/builtin/u2f_devices.go

```diff
@@ -192,6 +192,11 @@ var u2fDevices = []u2fDevice{
 		VendorIDPattern:  "311f",
 		ProductIDPattern: "4a2a",
 	},
+	{
+		Name:             "OneSpan DIGIPASS FX Series",
+		VendorIDPattern:  "1a44",
```

Comment:
for reviewers: https://www.usb.org/sites/default/files/vendor_ids073024.pdf confirms that 0x1a44 is OneSpan (also previously known as Vasco Digital Security and shows up in other USB Vendor ID lookups.

@jovasco I wasn't able to find a datasheet or windows drivers that would list the product IDs. I'm assuming that since you're at @onespan.com the information is correct, but it would be helpful if you could point at any documentation which confirms those.